sources/procfs.go

```diff
@@ -138,12 +138,12 @@ func (s *lustreSource) generateOSTMetricTemplates() error {
 			{"brw_size", "brw_size_megabytes", "Block read/write size in megabytes", s.gaugeMetric},
 			{"brw_stats", "pages_per_bulk_rw_total", pagesPerBlockRWHelp, s.counterMetric},
 			{"brw_stats", "discontiguous_pages_total", discontiguousPagesHelp, s.counterMetric},
-			{"brw_stats", "disk_io_now", diskIOsInFlightHelp, s.gaugeMetric},
+			{"brw_stats", "disk_io", diskIOsInFlightHelp, s.gaugeMetric},
 			{"brw_stats", "io_time_milliseconds_total", ioTimeHelp, s.counterMetric},
 			{"brw_stats", "disk_io_total", diskIOSizeHelp, s.counterMetric},
-			{"degraded", "degraded_now", "Binary indicator as to whether or not the pool is degraded - 0 for not degraded, 1 for degraded", s.gaugeMetric},
-			{"filesfree", "files_free_now", "The number of inodes (objects) available", s.gaugeMetric},
-			{"filestotal", "file_capacity", "The maximum number of inodes (objects) the filesystem can hold", s.gaugeMetric},
+			{"degraded", "degraded", "Binary indicator as to whether or not the pool is degraded - 0 for not degraded, 1 for degraded", s.gaugeMetric},
+			{"filesfree", "inode_free", "The number of inodes (objects) available", s.gaugeMetric},
```

Comment:
This might be an unpopular opinion, but I prefer `inodes_free` here. I interpret `inode_free` to mean whether a particular inode is free or not (which doesn't make sense), but that's just me.